opencraft/settings.py

```diff
@@ -292,6 +292,7 @@
 
 # Default github repository to pull code from
 DEFAULT_FORK = env('DEFAULT_FORK', default='edx/edx-platform')
+DEFAULT_EDX_PLATFORM_REPO_URL = 'https://github.com/{}.git'.format(DEFAULT_FORK)
```

Comment:
@smarnach Happy to see this definition move to the settings file! It felt out of place in the previous location.